logback/src/main/java/com/linecorp/armeria/common/logback/RequestContextExporterBuilder.java

```diff
@@ -117,9 +117,13 @@ private static AsciiString toHeaderName(CharSequence name) {
     void export(String mdcKey) {
         requireNonNull(mdcKey, "mdcKey");
 
-        final Optional<BuiltInProperty> opt = BuiltInProperty.findByMdcKey(mdcKey);
-        if (opt.isPresent()) {
-            builtIns.add(opt.get());
+        final List<BuiltInProperty> builtInPropertyList = BuiltInProperty.findByMdcKeyPattern(mdcKey);
+        if (!builtInPropertyList.isEmpty()) {
+            builtIns.addAll(builtInPropertyList);
+            return;
+        }
+
+        if (mdcKey.contains(BuiltInProperty.WILDCARD_STR)) {
             return;
```

Comment:
Just wondering if we need to log or throw an exception when the key contains `*` and there are no properties found for that.

Reply:
Since the document tells explicitly that it only supports built-in properties, it is better to tell it clearly if no matches are found.

Reply:
@masonshin Could you revert it, please? 🙇 
Have thought about it again and we can skip this case silently because it means including the properties matched by the pattern. (i.e there could be no match and we can skip to the next one.)

Reply:
I agree. Let me revert it.